### PR TITLE
Make default mappings optional

### PR DESCRIPTION
**Pull request by chadyred**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?   | no

Hello,

Firstly, thanks for your work. When I use this plugin I notice that it could be great when we override paths for mappings, to keep the default only if we don't define others in configure, in order to not mix what we expected to do in our own mapping VS the default one.

In fact, this is the first time I use it, maybe I misunderstand something :)

### CODE REVIEW

File: src/DependencyInjection/SynoliaSyliusGDPRExtension.php [hunk 1]
```diff
@@ -25,8 +25,8 @@ public function load(array $configs, ContainerBuilder $container): void
 
         $container->registerForAutoconfiguration(LoaderInterface::class)
             ->addTag('anonymization_loader');
-
         $mapping = $this->retrieveMappings($configs);
+        var_dump($mapping);
```
Comment (oallain) [inline src/DependencyInjection/SynoliaSyliusGDPRExtension.php]:
```suggestion
```

**Review by oallain — COMMENTED** [review]
Thanks for feedback and for this PR 🙂

**Comment by chadyred** [comment]
Your welcome ^^

**Comment by chadyred** [comment]
@oallain It is good for the typo ^^

**Comment by GalloisLuca** [comment]
Hello @chadyred, thanks for the PR ! I'm not sure if i understand well what you did. So, what I understand is that if someone make a new configuration path to override or to add configuration for an entity, every default configuration will be erased ? If that's so, I'm not sure that's a good thing to include in the plugin, not like this. We could make an option to decide if the user wants to keep the default configuration or not, but that's not the same dev.

**Comment by chadyred** [comment]
I understand, something like a trigger to "enabled" the default mappings or nor, sure ! I check that soon, have a nice week-end

**Review by GalloisLuca — DISMISSED** [review]

**Comment by GalloisLuca** [comment]
@chadyred Sounds good to me. If you can sign the license/cla with your account Florian Cellier that would be great :slightly_smiling_face:

**Comment by chadyred** [comment]
@GalloisLuca Thanks ! I tried to add the other email to my account but the licence/cla seems not to be validatedt with my other email. But no problem, I rebase and I have author with the first email address

**Review by GalloisLuca — APPROVED** [review]

**Comment by GalloisLuca** [comment]
@chadyred No probleme ! Thanks again for the contribution :smile:

@oallain Can you merge it plz ?

**Review by oallain — APPROVED** [review]

**Comment by oallain** [comment]
Thanks @chadyred  :rocket:

**Pull request merged**